components/menu-dropdown/__docs__/storybook-stories.jsx

```diff
@@ -134,6 +134,7 @@ const getDropdownPositioned = (props) => {
 		positionedDropdowns.push(
 			<div
 				className="slds-col slds-size_1-of-3"
+				key={`positioned-dropdoown-${position.replace(' ', '')}`}
```

Comment:
typo: `dropdoown`

Reply:
This is what I was talking about with my keyboard having messed up "I" and "O" keys that sometimes duplicate the inputs 😆